lib/mongoose_push_web/schemas/response/send_notification/generic_error.ex

```diff
@@ -0,0 +1,19 @@
+defmodule MongoosePushWeb.Schemas.Response.SendNotification.GenericError do
+  require OpenApiSpex
+  alias OpenApiSpex.Schema
+
+  OpenApiSpex.schema(%{
+    title: "Response.SendNotification.GenericError",
+    description: "Response schema for push notification request",
+    type: :object,
+    properties: %{
+      details: %Schema{
+        type: :string,
+        description:
+          "Short description of the encountered error. For development/debugging purposes only, " <>
+            "not for any client usage, as it may change in time without prior notice."
+      }
+    },
+    example: %{}
```

Comment:
We could add an example here (from tests?).

Reply:
Done.